test/data/TestData.scala

```diff
@@ -43,10 +51,12 @@ object TestData {
   val testAgentEORI: EORI                            = EORI("GB123456123444")
   val testRepresentativeAgentName: String            = "Agent name"
   val testRepresentativeImporterName: UserName       = UserName("ImporterName")
-  val testImporterManualAddress: Address             = Address("line 1", Some("line 2"), "City", Some("Region"), "GB", "AA11AA")
+
+  val testImporterManualAddress: Address =
+    Address("line 1", Some("line 2"), "City", Some("Region"), ukCountry, "AA11AA")
 
   val testAgentManualAddress: Address =
-    Address("line 1 agent", Some("line 2 agent"), "City agent", Some("Region agent"), "IT", "AA11AA")
+    Address("line 1 agent", Some("line 2 agent"), "City agent", Some("Region agent"), Country("IT", "Italy"), "AA11AA")
```

Comment:
CLARIFICATION: Postcode optional for non GB countries?  Does this effect this line, do we need an additional version or am I barking the wrong tree.